Dockerfile

```
@@ -14,7 +14,7 @@ RUN apk upgrade --no-cache -a && \
     clean-modules --yes && \
     npm cache clean --force
 
-FROM node:22.9.0-alpine3.20
+FROM node:22.9.0-alpine
```

```suggestion
FROM node:22.9.0-alpine
```

Should we then also rather use `node:22-alpine` instead to not be bound to 22.9 or is dependabot bumping those?

---

Dependabot is bunping those correctly usually